### ISSUE TITLE
RetroArch and cores use outdated 3ds library version

### ISSUE THREAD

**Issue by meepingsnesroms** [libretro/RetroArch#5799]
RetroArch uses an outdated version,  v1.1.0, of libctru(formery called ctrulib), the current version is v1.4.0(as of Nov 26 2017)

On compiling with the newest libctru you get error undefined function "GPUCMD_Finalize".

NOTE: This is not a bug but it should be fixed in the future, so I am logging it for reference.

**Comment by twinaphex** [comment]
Hi there @meepingsnesroms , does the current 3DS version of RetroArch nightly work for you? We had some people coming into #retroarch earlier today who said they would just get an ARM9 exception upon trying to run it, but I'm not sure which build they were actually using.

**Comment by meepingsnesroms** [comment]
I have only tested my build not the nightly, I did not bother to test the nightly since it is incomplete because the changes to mame2003 made it crash half way through the build, but there should be no code issues as I did not change devkitarm or its librarys as proposed here and no code was changed only assets.

Although I have gotten arm9 exceptions before when the last loaded core is no longer present and I load the salamander bootloader and it attempts to load a progarm that is not there.

Also they need to update there .core files that have the title ids in them or they will still be using invalid ones that just try to launch the bootloader because they all had title id 0xBAC00.
